### ISSUE TITLE
Problem with Help respond

### ISSUE THREAD

**Issue by meaz** [Ajnasz/hubot-respond#8]
If you do `hubot> help` you get:

```
delete respond to {a text} - Deletes respond respond_to
respond to {a text} with {value} - Creates a respond to respond_to and responds with value
list responds - Lists all responds
```
ALong with a lot of commands, as it is the way hubot-help works.
If you do `hubot> help respond` To get only the commands for hubot-respond, you get:

```
respond to {a text} with {value} - Creates a respond to respond_to and responds with value
list responds - Lists all responds
```
So as you can see `delete respond to {a text} - Deletes respond respond_to` disappeared

**Comment by Ajnasz** [comment]
Should be fixed, please check `hubot-respond@v1.2.1`

**Comment by meaz** [comment]
it does thanks!
